src/mmw/js/src/analyze/models.js

```diff
@@ -54,6 +54,33 @@ var AnalyzeTaskModel = coreModels.TaskModel.extend({
     }
 });
 
+var AnalyzeTaskCollection = Backbone.Collection.extend({
+    model: AnalyzeTaskModel
+});
+
+AnalyzeTaskCollection.getSingleton = function(App, aoi) {
+    if (!App.analyzeCollection) {
```

Comment:
Ah, we were deleting `analyzeModel` in the Draw controller. Updated that to deleting `analyzeCollection` in 028ed43